frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js

```diff
@@ -539,22 +552,22 @@ class PlaneController extends React.PureComponent<Props> {
     viewHandler?: ?Function,
   ): Function {
     return (...args) => {
-      if (skeletonHandler && volumeHandler) {
-        // Deal with both modes
-        const tool = Utils.enforce(getVolumeTool)(this.props.tracing.volume);
-        if (tool === VolumeToolEnum.MOVE) {
+      const tool = this.props.tracing.activeTool;
+      if (tool === AnnotationToolEnum.MOVE) {
+        if (viewHandler != null) {
+          viewHandler(...args);
+        } else if (skeletonHandler != null) {
```

Comment:
Does this mean that there is `skeletonHandler` functionality that should be executed in MOVE mode? I would've expected this to no longer be the case after this PR. Or will this only be cleaned up in the other PR?

Reply:
The second PR improves this a bit, but in the end it's still a bit mangled. I'll try to summarize it:

- This particular function (with the behavior that skeleton-related handlers are used as fallback when the move tool is active) is only used for keyboard-bindings thanks to the second PR. This means, hitting "c" in the move tool will create a new tree. This is probably up for debate, but it's quite compatible with the old behavior.
- With the new PR, there is still some entanglement of the move tool with other tools. For example, the `scroll` handler of the move tool is also responsible for changing the brush size (if `shift` is pressed). If two tools registered a `scroll` handler, this would be a conflict (which cannot be resolved using the active tool alone). Therefore, I went with avoiding these conflicts by implementing the bindings in the most generic tool. Luckily, this is only the case for very few handlers.

Reply:
Thank you for explaining! This seems to be a good solution for now :+1: